share/sysbuild/bootloader/Kconfig

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config SUPPORT_BOOTLOADER
+	bool
+	default y
+
+config SUPPORT_BOOTLOADER_MCUBOOT_ZEPHYR
+	bool
+	default y
+
+choice BOOTLOADER
+	prompt "Bootloader support"
+	default BOOTLOADER_NONE
+	depends on SUPPORT_BOOTLOADER
+
+config BOOTLOADER_NONE
+	bool "None"
+	help
+	  Do not Include a bootloader in the build
+
+config BOOTLOADER_MCUBOOT
```

Comment:
As I understand there is dedicated namespace for these Kconfig properties (and will never merge to with the application properties).
For instance this property does something else than CONFIG_BOOTLOADER_MCUBOOT of zephyr-rtos app.
I want to avoid confusion.

Reply:
I believe the help text can definitely be improved.

But whether we should invent two different setting names or have them identical as now depends a lot on how you look at this.

The Zephyr setting states:
https://github.com/zephyrproject-rtos/zephyr/blob/83d73f612607fa7f99ac639499e884cca9ba1500/Kconfig.zephyr#L682-L686

and my view on this is that this is also true for the sysbuild `BOOTLOADER_MCUBOOT` because it propagates this setting to the application so that the app is chain loaded by MCUboot, as well as devicetree is setup accordingly.
Not inside sysbuild itself, but in the images built by sysbuild.
Ref:
https://github.com/zephyrproject-rtos/zephyr/blob/70e05e2d2a379ba71d24430382a183706c865142/share/sysbuild/CMakeLists.txt#L33-L37

Of course sysbuild itself does one more thing which the application itself cannot do, and that is building and flashing of MCUboot in addition.

I believe having identical name for `BOOTLOADER_MCUBOOT` makes it easier for users to transition to sysbuild and understand how to build a sample with MCUboot, and that the small addition that sysbuild provides is not sufficient to justify two different names.